qiskit_nature/second_q/circuit/library/ansatzes/succd.py

```diff
@@ -106,6 +110,26 @@ def include_singles(self, include_singles: Tuple[bool, bool]) -> None:
         """Sets whether to include single excitations."""
         self._include_singles = include_singles
 
+    @property
+    def mirror(self) -> bool:
+        """Whether to include the symmetrically mirrored double excitations."""
+        return self._mirror
+
+    @mirror.setter
+    def mirror(self, mirror: bool) -> None:
+        """Sets whether to include the symmetrically mirrored double excitations."""
+        self._mirror = mirror
```

Comment:
I think this should also call `self._invalidate()` like the other setters do in the base UCC class since this will also alter the ansatz whereby it needs to signal it needs to be re-built according to the new configuration when its needed.